srml/executive/src/lib.rs

```diff
@@ -249,6 +249,12 @@ impl<
 
 			// increment nonce in storage
 			<system::Module<System>>::inc_account_nonce(sender);
+		} else {
+			// TODO TODO: does this can be wrongly called for just root extrinsic ? maybe we don't care
```

Comment:
Yes it is wrong: inherent are now refused.
* either we change validate_transaction to return an Option
* or we try to make Validatable operate on a structure that knows if it needs to be validated by modules